geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleHopDUnitTest.java

```diff
@@ -956,16 +935,16 @@ public void testClientMetadataForPersistentPrs() throws Exception {
       });
     }
 
-    LATCH.get().await(getTimeout().toMillis(), MILLISECONDS);
-
+    assertThat(LATCH.get().await(getTimeout().toMillis(), MILLISECONDS))
+        .as("Waiting for the latch").isTrue();
     AsyncInvocation<Integer> createServerOnVM3 =
-        vm3.invokeAsync(() -> createServer("disk", -1, 3, 4));
+        vm3.invokeAsync((SerializableCallableIF<Integer>) this::createServer);
     AsyncInvocation<Integer> createServerOnVM2 =
-        vm2.invokeAsync(() -> createServer("disk", -1, 3, 4));
+        vm2.invokeAsync((SerializableCallableIF<Integer>) this::createServer);
     AsyncInvocation<Integer> createServerOnVM1 =
-        vm1.invokeAsync(() -> createServer("disk", -1, 3, 4));
+        vm1.invokeAsync((SerializableCallableIF<Integer>) this::createServer);
     AsyncInvocation<Integer> createServerOnVM0 =
-        vm0.invokeAsync(() -> createServer("disk", -1, 3, 4));
+        vm0.invokeAsync((SerializableCallableIF<Integer>) this::createServer);
```

Comment:
More casts that can be avoided by using lambdas rather than method references.

Reply:
Same as above.

Reply:
I will swap them to lambdas...